design-tokens/build.js

```diff
@@ -36,11 +36,14 @@ const licenseHeader = fs.readFileSync(`config/license/LICENSE-HEADER.txt`, 'utf-
 const designTokenGroups = JSON.parse(
   fs.readFileSync(`${DESIGN_TOKENS_PATH}/$themes.json`, 'utf-8'),
 );
+const baseDesignTokenGroup = designTokenGroups.filter(
+  ({ group, name }) => group === 'Sonar' && name === 'base',
+)[0];
```

Comment:
NP:
```
const baseDesignTokenGroup = designTokenGroups.find(
  ({ group, name }) => group === 'Sonar' && name === 'base',
);
```

And maybe we should exit if we don't find it?

Reply:
Good catch, I'll update it.

> And maybe we should exit if we don't find it?

I don't think it's worth going through the hassle of handling these cases, it shouldn't happen, if it does, it's not gonna generate the expected files and/or break and that's fine.